pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -249,26 +210,23 @@ func (e ErrNetworkPolicyAppliedToUnsupportedGroup) Error() string {
 	return fmt.Sprintf("group %s/%s with IPBlocks or NamespaceSelector can not be used as AppliedTo", e.namespace, e.groupName)
 }
 
-func (n *NetworkPolicyController) processAppliedToGroupForNamespacedGroup(namespace, groupName string) (string, error) {
-	// Retrieve Group for corresponding entry in the AppliedToGroup.
-	g, err := n.grpLister.Groups(namespace).Get(groupName)
-	if err != nil {
-		// The Group referred to has not been created yet.
-		return "", nil
-	}
-	key := internalGroupKeyFunc(g)
-	// Find the internal Group corresponding to this Group
+func (n *NetworkPolicyController) createAppliedToGroupForNamespacedGroup(namespace, groupName string) (*antreatypes.AppliedToGroup, error) {
+	// Namespaced group uses NAMESPACE/NAME as the key of the corresponding internal group.
+	key := k8s.NamespacedName(namespace, groupName)
```

Comment:
I'm not sure if the name `internalGroupKeyFunc` still make sense since it is only applicable to ClusterGroup now.:
```
// internalGroupKeyFunc knows how to generate the key for an internal Group based on the object metadata
// of the corresponding ClusterGroup resource. Currently the Name of the ClusterGroup is used to ensure uniqueness.
func internalGroupKeyFunc(obj metav1.Object) string {
	return obj.GetName()
}
```
For better readability, we can either 1. use a type assertion and return different key for Namespaced Group and ClusterGroup 2. have separate key funcs for these resources

Reply:
It has been changed by https://github.com/antrea-io/antrea/pull/2438
https://github.com/antrea-io/antrea/blob/65b62cc645e83cce0a605a40422d307bf1767d4a/pkg/controller/networkpolicy/networkpolicy_controller.go#L1556-L1561